dotCMS/src/main/java/com/dotcms/rest/api/v1/site/SiteHelper.java

```diff
@@ -201,4 +205,34 @@ public String getSelectedSite(final List<Host> siteList, final String siteInSess
     	
     }
 
+	/**
+	 * Return the number of sites
+	 *
+	 * @return
+	 */
+	public long getSitesCount(final User user){
+    	return this.hostAPI.count(user, Boolean.FALSE);
+	}
+
+	/**
+	 * Return the current site
+	 *
+	 * @param req
+	 * @return
+	 */
+	public Host getCurrentSite(final HttpServletRequest req, final User user) {
+		try {
+			final HttpSession session = req.getSession();
+			String hostId = (String) session.getAttribute(com.dotmarketing.util.WebKeys.CMS_SELECTED_HOST_ID);
+			Host host = null;
```

Comment:
you can reduce it to

(null != hostId)?hostAPI.find(hostId, user, false):null;